docs/blog/0.32.md

```diff
@@ -0,0 +1,50 @@
+**---
+
+date: 2023-08-02
+image: https://github.com/rilldata/rill/assets/5587788/b30486f6-002a-445d-8a1b-955b6ec0066d
+
+---
+
+# Rill 0.32 -  Level Up Your Data Exploration 📈
+
+:::note
+⚡ Rill Developer is a tool that makes it effortless to transform your datasets with SQL and create fast, exploratory dashboards.
+
+To [try out Rill Developer, check out these instructions](../../install) and [let us know over on Discord](https://bit.ly/3bbcSl9) if you encounter any problems or have ideas about how to improve Rill Developer!
+:::
+
+
+## 🦆 DuckSQL Comes to Sources with a YAML editor! 
+
+DuckSQL support in source definition allows you to do more data wrangling during initial load.
+The YAML editor makes it easy to specify custom schemas, filters, and aggregations to get your ducks in a row before analysis.
+
+
```

Comment:
1. It seems a little confusing to talk about editing SQL in YAML
2. It would be nice to include a snippet or docs link (but seems we haven't updated the docs yet)
3. nit: Consider using "DuckDB SQL" instead of "DuckSQL" (the term doesn't show up in search)